### PR TITLE
Feat Auth and Theme

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added Following Features:
Theme using local-Storage
Authentication
Routes

### CODE REVIEW

File: src/frontend/pages/Explore/Explore.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.explore_wrapper{
+    display: flex;
+    width:100vw;
+    height:100%;
+    margin: 0.2rem 0;
+    box-sizing: border-box;
+    background-color: var(--bg2-color);
+}
+.explore_main_container{
+    display: flex;
```
Comment (manojkumar8400) [inline src/frontend/pages/Explore/Explore.css:10]: One suggestion - You can use class for repeating CSS properties like flex.

File: src/backend/controllers/AuthController.js [hunk 2]
```diff
@@ -60,6 +60,44 @@ export const signupHandler = function (schema, request) {
...
+//     );
+//   } catch (error) {
+//     return new Response(
+//       500,
+//       {},
+//       {
+//         error,
+//       }
+//     );
+//   }
```
Comment (manojkumar8400) [inline src/backend/controllers/AuthController.js]: Remove unnecessary comments.

**Review by manojkumar8400 — APPROVED** [review]
else code is good

**Review by bipin7yadav — APPROVED** [review]
good to go

**Pull request merged**